src/oca_github_bot/config.py

```diff
@@ -67,5 +68,11 @@ def func_wrapper(*args, **kwargs):
     "Hey, thanks for contributing! Proceeding to merge this for you.",
     "What a great day to merge this nice PR. Let's do it!",
 ]
+REBASE_BOT_INTRO_MESSAGES = [
+    "On my way to rebase this fine PR!",
+    "This PR looks fantastic, let's rebase it!",
+    "Hey, thanks for contributing! Proceeding to rebase this for you.",
+    "What a great day to rebase this nice PR. Let's do it!",
+]
```

Comment:
Not sure we need this for a simple reabase :)

Reply:
But it's fun, I like it.